From: Helmut Grohne <helmut@subdivi.de>
Date: Thu, 5 Sep 2024 17:20:14 +0200
Subject: Use AC_PATH_TOOL to locate python3-config

This helps finding the right python3-config when dealing with cross
compilation.
---
 configure.ac | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/configure.ac b/configure.ac
index aa64c45..ea6d47c 100644
--- a/configure.ac
+++ b/configure.ac
@@ -62,7 +62,7 @@ else
             ;;
     esac
     if test "x$__vk_tmp" = x; then
-        AC_PATH_PROG([PYTHON_CONFIG], [python2-config python-config], [no])
+        AC_PATH_TOOL([PYTHON_CONFIG], [python2-config python-config], [no])
         if test "x${PYTHON_CONFIG}" = xno; then
             __vk_tmp=python2-config
         fi
@@ -132,7 +132,7 @@ else
             ;;
     esac
     if test "x$__vk_tmp" = x; then
-        AC_PATH_PROG([PYTHON3_CONFIG], [python3-config], [no])
+        AC_PATH_TOOL([PYTHON3_CONFIG], [python3-config], [no])
         if test "x${PYTHON3_CONFIG}" = xno; then
             __vk_tmp=python3-config
         fi
